### PR TITLE
ECS-55 Fix internet gateway output 

### PR DESCRIPTION
**Pull request by mhennahane**
Another conditional code defensive change.  Should not affect any of our code - we don't use this output for anything downstream.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -93,5 +93,5 @@ output "natgw_ids" {
 }
 
 output "igw_id" {
-  value = "${aws_internet_gateway.mod.id}"
+  value = "${element(concat(aws_internet_gateway.mod.*.id, list("")), 0)}"
```
Comment (watchwithmike) [inline outputs.tf]: I do not understand what this is trying to achieve, it seems convoluted.  Is this necessary?

Comment (watchwithmike) [reply outputs.tf]: Would this be the same as saying `${element(coalescelist(aws_internet_gateway.mod.*.id, list("")), 0)}` ?

Comment (watchwithmike) [reply outputs.tf]: If that's the case, I'd prefer it if we could do a more explicit conditional if possible.

Comment (mhennahane) [reply outputs.tf]: It's done this way to match the code used upstream.   Line 104:

https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/outputs.tf

Comment (watchwithmike) [reply outputs.tf]: Ok, I don't need PRs if we're just merging in upstream changes.   I would probably do things differently but it's not significant enough to diverge.

**Comment by mhennahane** [comment]
Closing PR.  Adding "description", then merging both changes without PR, per MD comment that upstream changes can merge without PR.

**Pull request closed without merging**